### ISSUE TITLE
Process#kill: Unknown signal: EXIT (ArgumentError)

### ISSUE THREAD

**Issue by deepj** [oracle/truffleruby#1474]
This was originally caught in this Rack spec https://github.com/rack/rack/blob/master/test/spec_server.rb#L146 then https://github.com/rack/rack/blob/master/lib/rack/server.rb#L445.

There is no issue under MRI (Ruby 2.5.3). Something about `kill -0` command https://stackoverflow.com/a/14296353

To reproduce:

```ruby
Process.kill(0, Process.pid)
```

Error:

```
process.rb:1:in `kill': Unknown signal: EXIT (ArgumentError)
	from process.rb:1:in `each'
	from process.rb:1:in `kill'
	from process.rb:1:in `<main>'
```

**Comment by nirvdrum** [comment]
Thanks for reporting this. It looks like there's no signal with value 0. Ruby uses a special name of "EXIT" for it, but otherwise it doesn't have a name. Calling `kill` with a value of `0` for the signal performs permissions checks without actually sending a signal to the process. We currently use Java to send signals and the Java mechanism needs to be able to look up a signal by name, but there is no name. So, as far as I can tell, there's no way to send this special signal value from Java. We may have to just break out and make a POSIX call directly.

**Comment by nirvdrum** [comment]
Thanks again for the report. I've fixed the issue and added a spec for it in 063b05535764d6c697b20d4d5dfe50ca20609d2e.

**Comment by nirvdrum** [comment]
@deepj The fix for this should have shipped with the new 1.0.0 RC 11 release. Please comment here if you're still seeing a problem.
